tools/accuracy_checker/accuracy_checker/data_analyzer/detection_analyzer.py

```diff
@@ -44,31 +45,50 @@ def analyze(self, result: list, meta, count_objects=True):
             counter.update(data.labels)
 
         if count_objects:
-            self.object_count(result)
+            data_analyze['annotations_size'] = self.object_count(result)
 
         print_info('Total boxes {}'.format(all_boxes))
+        data_analyze['all_boxes'] = all_boxes
         label_map = meta.get('label_map', {})
 
         for key in counter:
             if key in label_map:
                 print_info('{name}: {value}'.format(name=label_map[key], value=counter[key]))
+                data_analyze[label_map[key]] = counter[key]
             else:
                 print_info('class_{key}: {value}'.format(key=key, value=counter[key]))
+                data_analyze['class_' + key] = counter[key]
 
         if size > 0:
+            diff_avg = diff_objects/size
+            objects_avg = all_boxes/size
+            width_avg = width/all_boxes
+            height_avg = height/all_boxes
+
             print_info(
-                'Average number of difficult objects (boxes) per image: {average}'.format(average=diff_objects/size)
+                'Average number of difficult objects (boxes) per image: {average}'.format(average=diff_avg)
             )
             print_info(
-                'Average number of objects (boxes) per image: {average}'.format(average=all_boxes/size)
+                'Average number of objects (boxes) per image: {average}'.format(average=objects_avg)
             )
             print_info(
                 'Average size detection object: width: {width}, '
-                'height: {height}'.format(width=width/all_boxes, height=height/all_boxes)
+                'height: {height}'.format(width=width_avg, height=height_avg)
             )
 
+            data_analyze['diff_avg'] = diff_avg
+            data_analyze['objects_avg'] = objects_avg
+            data_analyze['width_avg'] = width_avg
+            data_analyze['height_avg'] = height_avg
+
             if diff_objects > 0:
+                diff_width_avg = width_diff/diff_objects
+                diff_height_avg = height_diff / diff_objects
                 print_info(
                     'Average size difficult object: width: {width}, '
-                    'height: {height}\n'.format(width=width_diff/diff_objects, height=height_diff/diff_objects)
+                    'height: {height}\n'.format(width= diff_width_avg, height=diff_height_avg)
                 )
+                data_analyze['diff_width_avg'] = diff_width_avg
+                data_analyze['diff_height_avg'] = diff_height_avg
```

Comment:
please give more descriptive names for analysis keys
```suggestion
                data_analyze['average_width'] = diff_width_avg
                data_analyze['average_height'] = diff_height_avg
```